apps/admin/app/api/post/route.ts

```diff
@@ -65,7 +65,7 @@ const route = async (req: Request) => {
       return NextResponse.json(result);
     }
     case "getList": {
```

Comment:
by the way it's the only example of custom `case "getList":` would be useful to have at least a resource with it